internal/helm/values.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"helm.sh/helm/v3/pkg/chartutil"
+)
+
+func ToValues(v interface{}) (chartutil.Values, bool) {
+	switch c := v.(type) {
+	case chartutil.Values:
+		return c, true
+	case map[string]interface{}:
+		return c, true
+	default:
+		return nil, false
+	}
+}
+
+func OverrideValuesWithValues(dst chartutil.Values, src chartutil.Values) {
```

Comment:
can we use `chartutil.CoalesceTables` instead?

Reply:
Sure. Thank you.